packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -20,38 +20,49 @@ import { ISystemConfigV0 } from "scripts/interfaces/ISystemConfigV0.sol";
 import { console2 as console } from "forge-std/console2.sol";
 
 library ChainAssertions {
-    /// @notice Asserts the correctness of an L1 deployment
+    /// @notice Asserts the correctness of an L1 deployment. This function expects that all contracts
+    ///         within the `prox` ContractSet are proxies that have been setup and initialized.
     function postDeployAssertions(
-        Types.ContractSet memory prox,
-        DeployConfig cfg,
-        uint256 l2OutputOracleStartingTimestamp,
-        Vm vm
+        Types.ContractSet memory _prox,
```

Comment:
should this be renamed to `_contracts` to match the other functions?

Reply:
I kept this named `_prox` because this function, unlike the others, _does_ rely on this `ContractSet` being proxies. See the doc comment above